tools_and_services/setting_up_windows_for_rails.md

```diff
@@ -0,0 +1,170 @@
+# Setting up your Windows 11 machine for running rails projects
+
+## Prerequisites
+
+- Windows 11
+- VS Code
+
+## 1. Install Ubuntu in wsl
+
+Open power shell and run the following command:
+
+```powershell
+wsl --install -d Ubuntu
+```
+
+The command will eventually will tell you to restart your machine. Do that.
+
+After the restart the installation will continue, and eventually it will prompt you for a username and password for your new linux user.\
+With the credentials in place, you will be launched into a linux terminal.
+
+## 2. Install ZSH and Oh-my-zsh
+
+```sh
+sudo apt -y install zsh
+chsh -s $(which zsh)
+```
+
+Open a new terminal window:
+
+1. Press the "⋁" button in the top of the terminal window. Then select "Ubuntu".
```

Comment:
What is this? :D

Reply:
![billede](https://github.com/abtion/guidelines/assets/1164553/4bb50c7c-3423-4c4c-9812-b5a8f66a1288)

I should perhaps have added a screenshot. But I was not writing the guide on the same machine as I was going through the process on